templates/books-show.html

```diff
@@ -0,0 +1,36 @@
+<h2>{{bsc.book.title}}</h2>
+<hr> 
+<div class="row">
+	<div class="col-xs-6">
+		<img ng-src="{{bsc.book.image}}" class="img-responsive">
+	</div>
+	<div class="col-xs-6">
+		<p><strong>Author:</strong> {{bsc.book.author}}</p>
+		<p><strong>Release Date:</strong> {{bsc.book.releaseDate}}</p>
+		<br>
+
+		<!-- buttons to edit/delete -->
+		<a href="#" ng-click="editForm=!editForm" class="btn btn-default">Edit <span class="glyphicon glyphicon-pencil"></span></a>&nbsp;
```

Comment:
why do you have an nbsp at the end of this line? It looks like a typo rather than intentional; if it's intentional, it belongs on the next line.